Corgibytes.Freshli.Cli.Test/ProgramTest.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.IO;
+using Corgibytes.Freshli.Cli.Test.Common;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Corgibytes.Freshli.Cli.Test
+{
+    public class ProgramTest : FreshliTest
+    {
+        public ProgramTest(ITestOutputHelper output) : base(output) {}
+
+        [Fact]
+        public void Validate_Main_loglevel_debug()
+        {
+            StringWriter stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+            Program.Main(new string [] {"--loglevel", "Debug" });
+            stringWriter.ToString().Should().Contain("DEBUG|Microsoft.Extensions.Hosting.Internal.Host:0|Hosting stopping");
+        }
+
+        [Fact]
+        public void Validate_Main_loglevel_info()
+        {
+            StringWriter stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+            Program.Main(new string [] {"--loglevel", "Info" });
+            stringWriter.ToString().Should().NotContain("DEBUG|Microsoft.Extensions.Hosting.Internal.Host:0|Hosting stopping");
```

Comment:
Why testing `NotContain`?

Reply:
This test is making sure that the logger does not produce any `DEBUG` messages when the log level is set to `INFO`.